### ISSUE TITLE
Return current strategy

### ISSUE THREAD

**Issue by JKetelaar** [Parabot/Parabot#241]

**Comment by AlexanderBielen** [comment]
Perhaps for a strategy to "unprovide" itself? However, both the [Script](https://github.com/Parabot/Parabot/blob/master/src/main/java/org/parabot/environment/scripts/Script.java) and [Framework](https://github.com/Parabot/Parabot/blob/master/src/main/java/org/parabot/environment/scripts/framework/Frameworks.java) class do not support that. Could be useful in some cases, but not sure if it's much needed.

**Comment by JKetelaar** [comment]
I'd like to see a case where this is useful, to know at what strategy we're at

**Comment by Shadowrs** [comment]
@JKetelaar I added this to my internal version - was very useful for debugging scripts (when running in real time) that hung in certain places by painting the info on screen. I had to add a flag to temporarily allow access in the `ClassRedirect` to get the class' real name since its in the parabot dir. 

```
    public static String getSimpleName(Class<?> c) {
        if (!Script.SKIP_SECRURITY && c.getName().contains("parabot")) {
            return "java.lang.String";
        }
        return c.getSimpleName();
    }```

**Comment by JKetelaar** [comment]
Good call, maybe we could make a debug out of it. I see the use of it now.
